MoltenVK/MoltenVK/Vulkan/mvk_datatypes.mm

```diff
@@ -759,7 +759,7 @@ MVK_PUBLIC_SYMBOL MTLTextureUsage mvkMTLTextureUsageFromVkImageUsageFlags(VkImag
     }
     if ( mvkAreFlagsEnabled(vkImageUsageFlags, VK_IMAGE_USAGE_DEPTH_STENCIL_ATTACHMENT_BIT) ) {
         mvkEnableFlag(mtlUsage, MTLTextureUsageRenderTarget);
-        mvkDisableFlag(mtlUsage, MTLTextureUsagePixelFormatView);        // Clears bit. Do this last.
+        mvkEnableFlag(mtlUsage, MTLTextureUsagePixelFormatView);
```

Comment:
Should this always be enabled...or only for the stencil component of a D/S format...and only when the new `_device->_pMetalFeaturesstencilViews` flag is enabled?

Reply:
I debated whether or not to do that. I really wanted to do that, but I thought in the interests of simplicity that it might be better not to. It would require us to pass the pixel format and the device, respectively. I agree it would be more correct.

Reply:
Yeah...it's a PITA. But...as you say...more correct.  :^(

See for instance how format translation is handled. There is a basic static version as `mvkMTLPixelFormatFromVkFormat()` in `mvk_datatypes.h`...but all calls go through `MVKDevice::mtlPixelFormatFromVkFormat()`...all for the single `MTLPixelFormatDepth24Unorm_Stencil8` format.

At least `mvkMTLTextureUsageFromVkImageUsageFlags()` is only called from one place.

Reply:
BTW...you can't pass the `MVKDevice` to `mvkMTLTextureUsageFromVkImageUsageFlags()` because `mvk_datatypes.h` is a public Vulkan-like runtime C API...and knows nothing of C++ objects.

Better to delegate to the device...which can use the static method and then add any customization.

Reply:
Thinking about this further...I'm not so sure you'd need to change `mvkMTLTextureUsageFromVkImageUsageFlags()` to pass the format. You should be able to leave `mvkMTLTextureUsageFromVkImageUsageFlags()` where it was...and move your test for the `stencilViews` flag to a new `MVKDevice` method.